stdlib/database/h2/src/main/ballerina/h2/endpoint.bal

```diff
@@ -16,20 +16,14 @@
 
 import ballerina/sql;
 
-# The Client endpoint configuration for h2 databases.
+# The Client endpoint configuration for the in-memory mode of h2 databases.
 #
-# + host - The host name of the database to connect (in case of server based DB)
-# + path - The path of the database connection (in case of file based DB)
-# + port - The port of the database to connect (in case of server based DB)
 # + name - The name of the database to connect
 # + username - Username for the database connection
 # + password - Password for the database connection
 # + poolOptions - Properties for the connection pool configuration. Refer `sql:PoolOptions` for more details
```

Comment:
Don't we need to update the documentation to reflect the changes? Where we mention the poolOptions, dbOptions in the record?